model/schema/expression.yaml

```diff
@@ -44,104 +44,255 @@ classes:
 
   ExpressionExperiment:
     is_a: SubmittedObject
+    abstract: true
     description: >-
-      Defined by the gene of interest, the specimen, the assay, the reagents
-      (Antibody, Probe), and the reference. It groups ExpressionAnnotations.
+      The experimental design used to determine the expression of some entity,
+      defined by the entity_assayed, the specimen, the assay, the reagents
+      (e.g., Antibody, Probe), the environmental conditions, and the reference.
+      It groups ExpressionAnnotations; it may have its own notes.
     notes: >-
-      Specimens are defined by their AGM and/or alleles (or gel lane at MGI).
+      Specimens are defined by their AGM (or gel lane at MGI).
+      This entity is originally specific to WB and MGI.
     slots:
+      - curie
```

Comment:
curie, mod_entity_id, mod_internal_id, and data_provider are all inherited from SubmittedObject so don't need to be defined here